input/europe/eu_reactors_pris.csv

```diff
@@ -12,8 +12,6 @@ Bulgaria,KOZLODUY-3,PWR,408,79,26,Permanent Shutdown,KOZNPP,19731001,1973,198012
 Bulgaria,KOZLODUY-4,PWR,408,79,26,Permanent Shutdown,KOZNPP,19731001,1973,19820425,,,19820517,19820620,20061231,
 Bulgaria,KOZLODUY-5,PWR,953,184,61,Operational,KOZNPP,19800709,1980,19871105,,,19871129,19881223,,86
 Bulgaria,KOZLODUY-6,PWR,953,184,61,Operational,KOZNPP,19820401,1982,19910529,,,19910802,19931230,,89.1
-Bulgaria,BELENE-1,PWR,953,184,61,Cancelled Constr.,KOZNPP,19870101,1987,,,,,,20120328,
```

Comment:
Definitely makes sense. Maybe later I'll try to come up with a filter in the input generation file to filter these out!